src/dr_prettify.py

```diff
@@ -64,7 +64,12 @@ def pretty_ip_protocol(p):
 
 def pretty_source_vport(regc0):
     source_vport = (int(regc0, 16) >> 16) - 1
-    source_vport = "(0x%x)," % (source_vport-1 if source_vport != 0 else source_vport)
+    if source_vport == 0:
+        source_vport = "pf"
+    elif source_vport >= 4:
+    #the base sf index for 1 pf is 2 + sriov_totalvfs
+    #cat /sys/bus/pci/devices/0000\:08\:00.0/sriov_totalvfs
+        source_vport = "sf%d" % (source_vport - 4)
```

Comment:
1. Could you check if the output port start sf0 or sf1?
2. Let use MARCO instead of 4 since it's fixed sometimes, we can change marco easier later.

Reply:
Checked, the both the output port and in port start from sf0 with this patch.